demo/px-data-grid-custom-renderer-demo.html

```diff
@@ -0,0 +1,251 @@
+<!-- Common demo imports -->
+<link rel="import" href="../../px-demo/px-demo-header.html" />
+<link rel="import" href="../../px-demo/px-demo-api-viewer.html" />
+<link rel="import" href="../../px-demo/px-demo-footer.html" />
+<link rel="import" href="../../px-demo/px-demo-configs.html" />
+<link rel="import" href="../../px-demo/px-demo-props.html" />
+<link rel="import" href="../../px-demo/px-demo-interactive.html" />
+<link rel="import" href="../../px-demo/px-demo-component-snippet.html" />
+<link rel="import" href="../../px-demo/px-demo-code-editor.html" />
+
+<!-- Imports for this component -->
+<link rel="import" href="../../px-demo/css/px-demo-styles.html" />
+<link rel="import" href="../px-data-grid.html" />
+
+<link rel="import" href="../px-data-grid-renderer-mixin.html">
+<link rel="import" href="../px-icon-set/px-icon.html">
+
+<dom-module id="px-data-grid-icon-renderer">
+  <template>
+    <style>
+      :host {
+        display: flex;
+        align-items: center;
+        flex-grow: 1;
+      }
+    </style>
+
+    <px-icon icon="[[value]]"></px-icon>
+  </template>
+  <script>
+    {
+      class DataGridIconRenderer extends Predix.DataGridRendererMixin(Polymer.Element) {
+        static get is() {
+          return 'px-data-grid-icon-renderer';
+        }
+      }
+
+      customElements.define(DataGridIconRenderer.is, DataGridIconRenderer);
+
+      /**
+       * @namespace Predix
+       */
+      window.Predix = window.Predix || {};
+      Predix.DataGridIconRenderer = DataGridIconRenderer;
+    }
+  </script>
+</dom-module>
+
+<!-- Demo DOM module -->
+<dom-module id="px-data-grid-custom-renderer-demo">
+  <template>
+    <custom-style>
+      <style include="px-demo-styles" is="custom-style"></style>
+    </custom-style>
+
+    <!-- Header -->
+    <px-demo-header
+        module-name='Custom grid renderer'
```

Comment:
just 'custom rendered', the 'grid' word is misleading

Reply:
Done.